### DISCUSSION TITLE
Unable to Resolve Complex GitHub Conflicts via Web Editor

### DISCUSSION THREAD

**Discussion by rajveer43** [community/community#68577] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area
Issue

### Body

![image](https://github.com/community/community/assets/64583161/b34ff0e0-b971-43b6-b870-ee31038d24e7)

## Problem Description
I am encountering a problem with resolving complex conflicts in a GitHub pull request. I have tried various potential fixes, but none have worked so far.

## Steps Taken
- Merging with rebase.
- Creating a clone branch for the issue.
- Creating a new pull request.
- Received a message stating that conflicts are too complex to solve in a web editor.

## Additional Fixes Attempted
- Updating the code editor.
- Trying a different code editor.
- Deleting and restoring the branch.
- Consulted GitHub documentation.
- Two more unspecified attempts.

## Request for Assistance
I am seeking assistance and guidance on how to resolve these complex conflicts and successfully complete the pull request. Any help or suggestions would be greatly appreciated.

**Comment by vishalkrishna8** [comment, marked as answer]
Hi @rajveer43 ,
You can try the following steps:

1.Ensure Local Repository is Up to Date: Before attempting to resolve conflicts, ensure that your local repository is up to date with the latest changes from the remote repository. You can use the following commands:

`git checkout main  # Switch to the main branch
git pull origin main  # Pull the latest changes from the remote repository
git checkout your-feature-branch  # Switch back to your feature branch
`

2.Rebase Your Branch: Instead of merging, try rebasing your branch onto the main branch. This can help you integrate the latest changes more smoothly and resolve conflicts incrementally. Use the following commands:

`git rebase main`

3.Consider creating a new pull request with your changes. Sometimes, starting fresh can help resolve conflicts.

4.If GitHub states the conflicts are too complex for the web editor, try resolving them locally on your computer using a code editor

Let me know if you need any further information.

**Reply by rajveer43** [reply]
thank you so much there were 500+ conflicted files!

**Reply by Ranaba123** [reply]
thanks

**Reply by designsixthsense** [reply]
how can we know that what are the conflicts in the code editor?

**Reply by Crparedes** [reply]
> how can we know that what are the conflicts in the code editor?

From [GitHub docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line):

> To see the beginning of the merge conflict in your file, search the file for the conflict marker <<<<<<<. When you open the file in your text editor, you'll see the changes from the HEAD or base branch after the line <<<<<<< HEAD. Next, you'll see =======, which divides your changes from the changes in the other branch, followed by >>>>>>> BRANCH-NAME. In this example, one person wrote "open an issue" in the base or HEAD branch and another person wrote "ask your question in IRC" in the compare branch or branch-a.
